.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [2.7, "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.11"]
```

Comment:
Definitely makes sense to drop 2.7, but does it make sense to _only_ test on 3.11? https://www.python.org/downloads/ says that 3.8, 3.9, 3.10 and 3.11 are all active releases.